src/components/views/messages/TextualBody.js

```diff
@@ -131,7 +131,8 @@ module.exports = React.createClass({
                     links.push(node);
                 }
             }
-            else if (node.tagName === "PRE" || node.tagName === "CODE") {
+            else if (node.tagName === "PRE" || node.tagName === "CODE" ||
```

Comment:
I thought of using `else if (["BLOCKQUOTE", "PRE", "CODE"].indexOf(node.tagName) > -1) {` here since it's a bit shorter, but I guess it would take a pretty big hit in performance